pyramid_openapi3/__init__.py

```diff
@@ -35,18 +35,25 @@ def wrapper_view(context, request):
             open_request = PyramidOpenAPIRequest(request)
             request.openapi_validated = settings['request_validator'].validate(open_request)
 
-            if request.openapi_validated.errors and getattr(request.registry, "_pyramid_openapi3_validation_view_name", None):
-                # handle request errors
-                view_name = request.registry._pyramid_openapi3_validation_view_name
-                response = render_view_to_response(context, request, name=view_name, secure=False)
-            else:
-                # Do the view
-                response = view(context, request)
-
-                # Validate response and raise if an error is found
-                open_response = PyramidOpenAPIResponse(response)
-                result = settings['response_validator'].validate(request=open_request, response=open_response)
-                result.raise_for_errors()
+            try:
+                if request.openapi_validated.errors and getattr(request.registry, "_pyramid_openapi3_validation_view_name", None):
+                    # handle request errors
+                    view_name = request.registry._pyramid_openapi3_validation_view_name
+                    response = render_view_to_response(context, request, name=view_name, secure=False)
```

Comment:
Are you sure we want secure=False here?  It's a crazy edge case to register a view via ``config.pyramid_openapi3_validation_error_view()`` that is protected by a permission, but it'd be surprising to me if I did register a view with a permission that got used for this purpose and the permission was ignored.  Leaving it unsecured is probably ok (and even desirable), but we should mention it in the docs; alternately we could omit secure=False and document that if there is a default view permission registered via ``config.set_default_permission`` that the view should be registered as permission=pyramid.security.NO_PERMISSION_REQUIRED (less desirable, I suppose).

Reply:
Good point, I'll make a note to document it.